invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -21,32 +21,34 @@
         "access_right": {
           "type": "keyword"
         },
-        "$schema": {
-          "type": "keyword",
-          "index": false
-        },
-        "recid": {
-          "type": "keyword"
-        },
-        "title": {
-          "type": "text",
-          "copy_to": "suggest_title"
-        },
-        "suggest_title": {
-          "type": "completion"
-        },
-        "description": {
-          "type": "text"
-        },
-        "owners": {
-          "type": "integer"
-        },
-        "keywords": {
-          "type": "keyword"
+        "additional_descriptions": {
+          "type": "object",
```

Comment:
normal: It will be important to think if we want "object" or "nested" for these types.

Reply:
As object they are a bit more performant AFAIK, however is true that values are flatten and might produce false positives. I see `type` and `lang` will potentially be part of the filters (classic checkbox aggregators for example), however I dont see how a title might have different types for different languages. Does this makes sense in your use cases?

Since we have the mayor `description` field which will be shown these `additional_descriptions:description` might just be there for search purposes.

I would prefer to have `object` since its easier to work with (for queries, for example, Invenio query parser does not support nested queries yet, see [issue](https://github.com/inveniosoftware/invenio-search/issues/155))